docs/topics/mtr-rn-resolved-issues-1-2-1.adoc

```diff
@@ -0,0 +1,14 @@
+// Module included in the following assemblies:
+//
+// * docs/release-notes-mtr/mtr_release_notes-1.0/master.adoc
+
+:_content-type: REFERENCE
+[id="mtr-rn-resolved-issues-1-2-1_{context}"]
+= Resolved issues
+
+
+.CVE-2023-44487 netty-codec-http2: HTTP/2
+
+Multiple HTTP/2 enabled web servers are vulnerable to a DDoS attack (Rapid Reset Attack). The HTTP/2 protocol allows a denial of service (server resource consumption) because request cancellation can reset many streams quickly. link:https://issues.redhat.com/browse/WINDUP-4056[(WINDUP-4056)]
+
+For a complete list of all issues resolved in this release, see the list of link:https://issues.redhat.com/browse/WINDUP-3952?filter=12422644[MTR 1.2.1 resolved issues] in Jira.
```

Comment:
@PhilipCattanach ,
This filter also brings https://issues.redhat.com/browse/WINDUP-3952
which is the 1.1 CVE. It is targeted to 1.2.1, but I think it should be